route/v1/types.go

```diff
@@ -70,7 +70,23 @@ type RouteSpec struct {
 	// chosen.
 	// Must follow DNS952 subdomain conventions.
 	Host string `json:"host" protobuf:"bytes,1,opt,name=host"`
-	// Path that the router watches for, to route traffic for to the service. Optional
+	// subdomain is a DNS subdomain that is requested within the ingress controller's
```

Comment:
From the WIP [router implementation](https://github.com/openshift/origin/pull/22304):

> subdomain must conform to DNS 952 subdomain conventions

Should that be made explicit here?

Reply:
Is dns subdomain enough?  I dunno how many users actually look up rfcs

Reply:
wfm